cypress/support/pages/home.pageObject.js

```diff
@@ -1,10 +1,19 @@
 import PageObject from '../PageObject';
 
 class homePageObject extends PageObject {
-  url = '/#/';
+  url = '/';
 
   get usernameLink() {
-    return cy.getByDataCy('/profile/');
+    return cy.getByDataCy('/profile');
+  }
+
+  assertUsernameLink(username) {
+    this.usernameLink.should('contain', username);
+  }
+
+  assertLogOut() {
+    cy.getByDataCy('/user/register').should('exist');
+    cy.getByDataCy('/user/login').should('exist');
```

Comment:
The data-cy attribute value should be more descriptive, consider changing '/user/login' to 'login-link'